### ISSUE TITLE
Netbox return hostname instead of IP

### ISSUE THREAD

**Issue by cdot65** [tbotnz/netpalm-admin#15]
Could we get the dynamic inventory script to return the hostname instead of IP address?

Makes the visual list selection a little more intuitive than seeing IP addresses

**Comment by tbotnz** [comment]
Yes, will add it. Are you not the type of guy to refer to his friends by their IP address?

**Comment by cdot65** [comment]
Are you not???

**Comment by cdot65** [comment]
looking for a much needed update @tbotnz
